bpf/go_nethttp.h

```diff
@@ -214,6 +225,57 @@ int uprobe_readRequestReturns(struct pt_regs *ctx) {
     return 0;
 }
 
+// Handles finding the connection information for http2 servers in grpc
+SEC("uprobe/http2Server_processHeaders")
+int uprobe_http2Server_processHeaders(struct pt_regs *ctx) {
+    void *sc_ptr = GO_PARAM1(ctx);
+    void *frame = GO_PARAM2(ctx);
+    bpf_dbg_printk("=== uprobe/http2Server_processHeaders sc %lx === ", sc_ptr);
+    go_addr_key_t g_key = {};
+    go_addr_key_from_id(&g_key, sc_ptr);
+
+    tp_info_t tp = {0};
+
+    process_meta_frame_headers(frame, &tp);
+
+    if (valid_trace(tp.trace_id)) {
+        bpf_dbg_printk("found valid traceparent in http2 headers");
+        bpf_map_update_elem(&http2_server_requests_tp, &g_key, &tp, BPF_ANY);
+    }
+
+    return 0;
+}
+
+SEC("uprobe/readContinuedLineSlice")
+int uprobe_readContinuedLineSliceReturns(struct pt_regs *ctx) {
+    bpf_dbg_printk("=== uprobe/proc readContinuedLineSlice returns === ");
+
+    void *goroutine_addr = GOROUTINE_PTR(ctx);
+    u64 len = (u64)GO_PARAM2(ctx);
+    u8 *buf = (u8 *)GO_PARAM1(ctx);
+
+    if (len >= 68) {
```

Comment:
any chance we could slap a name here instead of a magic number? It's not clear what 68 means without any further context.

Reply:
100%, I forgot to update that.